src/demo/app.ts

```diff
@@ -1,63 +1,54 @@
 import {addSpawnListeners, createApp, createWindow} from 'openfin-service-tooling/spawn';
 
 import * as ofnotes from '../client/index';
```

Comment:
If we are going to be doing named imports for all of the methods we probably don't need this import here anymore?

Reply:
Agree in general, want to get rid of any remaining `import *`'s that we have - though we do still use this one to pin the client API to the window, which is pretty handy...

Reply:
Oh yea, I'd forgotten about that part. Fair enough.